packages/core/src/tweening/interpolationFunctions.ts

```diff
@@ -66,11 +71,27 @@ export function deepLerp<
  * @returns A value matching the structure of from and to.
  */
 export function deepLerp<T>(from: T, to: T, value: number): T;
-export function deepLerp(from: any, to: any, value: number): any {
+export function deepLerp<T>(
+  from: T,
+  to: T,
+  value: number,
+  suppressWarnings: boolean,
+): T;
+export function deepLerp(
+  from: any,
+  to: any,
+  value: number,
+  suppressWarnings = false,
+): any {
   if (value === 0) return from;
   if (value === 1) return to;
 
-  if (typeof from === 'undefined' || typeof to === 'undefined') {
```

Comment:
This was letting `null` slip through as well, which caused issues further down the function.